Components/Services/Installments/InstallmentsPaymentBuilderService.php

```diff
@@ -8,12 +8,35 @@
 
 namespace SwagPaymentPayPalUnified\Components\Services\Installments;
 
+use Shopware\Components\Routing\RouterInterface;
+use Shopware_Components_Snippet_Manager as SnippetManager;
+use SwagPaymentPayPalUnified\Components\DependencyProvider;
+use SwagPaymentPayPalUnified\PayPalBundle\Components\SettingsServiceInterface;
 use SwagPaymentPayPalUnified\Components\PaymentBuilderParameters;
 use SwagPaymentPayPalUnified\Components\Services\PaymentBuilderService;
 use SwagPaymentPayPalUnified\PayPalBundle\Components\SettingsTable;
 
 class InstallmentsPaymentBuilderService extends PaymentBuilderService
 {
+
+    /**
+     * @var DependencyProvider
+     */
+    private $dependencyProvider;
+
+    public function __construct(
+        RouterInterface $router,
+        SettingsServiceInterface $settingsService,
+        SnippetManager $snippetManager,
+        dependencyProvider $dependencyProvider
```

Comment:
```suggestion
        DependencyProvider $dependencyProvider
```
There is a typo. This will work currently, but fail with PHP 8 🤔

Reply:
I will fix that :blush: